client/packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -138,7 +133,11 @@ export const useDialog = (dialogProps?: DialogProps): DialogState => {
     return (
       <BasicModal
         open={open}
-        onClose={handleClose}
+        onClose={reason => {
```

Comment:
nice! 😁 
I think it would be good to have it controlled by a prop, so that users of the hook could choose. The mui Modal has a few `disable..` props, to be consistent you could perhaps add a `disableBackdropClick` prop